browser/brave_news/brave_news_tab_helper.cc

```diff
@@ -71,8 +71,10 @@ BraveNewsTabHelper::GetAvailableFeeds() {
 bool BraveNewsTabHelper::IsSubscribed(const FeedDetails& feed_details) {
   auto* publisher = controller_->publisher_controller()->GetPublisherForFeed(
       feed_details.feed_url);
-  return publisher && publisher->user_enabled_status ==
-                          brave_news::mojom::UserEnabled::ENABLED;
+  return publisher &&
```

Comment:
How about adding comments like below for example? I'm always confusing :)  
```
if (!publisher)
  return false;

// User chosen feed. It doesn't have enabled/disabled state.
// When it exists, it means subscribed. If not, publisher conttroller doesn't have it.
if (publisher->type == brave_news::mojom::PublisherType::DIRECT_SOURCE)
  return true;

// User can enable/disable publishers from default-provided feeds.
return publisher->user_enabled_status == brave_news::mojom::UserEnabled::ENABLED;
```

Reply:
Good call!